src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/BootableVolumeList.tsx

```diff
@@ -147,12 +153,14 @@ const BootableVolumeList: FC<BootableVolumeListProps> = ({
         <Tbody>
           {sortedData?.map((bs) => (
             <BootableVolumeRow
-              key={bs?.metadata?.name}
+              key={getName(bs)}
               bootableVolume={bs}
-              activeColumnIDs={Object.values(activeColumns)?.map((col) => col?.id)}
+              activeColumnIDs={activeColumns?.map((col) => col?.id)}
               rowData={{
-                bootableVolumeSelectedState: [bootableVolumeSelected, setBootableVolumeSelected],
-                preference: preferences[bs?.metadata?.labels?.[DEFAULT_PREFERENCE_LABEL]],
+                bootableVolumeSelectedState: !displayShowAllButton
+                  ? selectedBootableVolumeState
+                  : [selectedBootableVolume, onSelectVolume],
+                preference: preferencesMap[bs?.metadata?.labels?.[DEFAULT_PREFERENCE_LABEL]],
```

Comment:
We should add a `getLabel(<label name>)` function. I'm sure we do this all over the place.

Reply:
you're right, execpt we do have this function already changed, Thanks :)